functions.php

```diff
@@ -88,6 +88,9 @@ function html5wp_pagination()
 add_action('init', 'create_post_type_projects');
 add_action('init', 'create_post_type_grantees');
 
+add_post_type_support( 'team', 'thumbnail' );
+add_post_type_support( 'videos', 'thumbnail' );
```

Comment:
Is this necessary when we are using a custom field for the thumbnails?

Reply:
that was the code to add featured images but I don't think we need it anymore since we are using custom image fields

Reply:
Yes, that was my impression too!